go/test/endtoend/tabletmanager/tablegc/tablegc_test.go

```diff
@@ -258,19 +299,11 @@ func TestEvac(t *testing.T) {
 	}
 
 	time.Sleep(10 * time.Second)
```

Comment:
Nit, but can we make this a variable ~ `createTableTimeStampOffsetSecs` and then use that everywhere we are doing `time.Sleep` in relation to that variable? We can then use this throughout:
```
time.Sleep(createTableTimeStampOffsetSecs * time.Second)
time.Sleep((createTableTimeStampOffsetSecs / 2) * time.Second)
time.Sleep((createTableTimeStampOffsetSecs * 3) * time.Second)
...
```

This way it's easier to tie all the timing behavior together and adjust it holistically as needed by increasing that one variable.

Reply:
All time values now parameterized